src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -62,12 +64,26 @@ public static class Vision {
 		public static double standardDeviationVisionAngle = 0.1;
 		public static int selectedTagId = 1;
 		public static final int[] portNumbers = {5800,5801,5802,5803,5804,5805};
-		public static final Pose3d apriltagLocation = new Pose3d(new Translation3d(5, 96.4, 0), new Rotation3d(0, 0, Math.PI));
-		static List<AprilTag> apriltags = new ArrayList<>(5) ;
+		public static final Pose3d redApriltagLocationId1 = new Pose3d(new Translation3d(15.513558, 1.071626, 0), new Rotation3d(0, 0, Math.PI));
+		public static final Pose3d redApriltagLocationId2 = new Pose3d(new Translation3d(15.513558, 2.748026, 0), new Rotation3d(0, 0, Math.PI));
+		public static final Pose3d redApriltagLocationId3 = new Pose3d(new Translation3d(15.513558, 4.424426, 0), new Rotation3d(0, 0, Math.PI));
+		public static final Pose3d blueApriltagLocationId1 = new Pose3d(new Translation3d(1.02743, 4.424426, 0), new Rotation3d(0, 0, Math.PI));
+		public static final Pose3d blueApriltagLocationId2 = new Pose3d(new Translation3d(1.02743, 2.748026, 0), new Rotation3d(0, 0, Math.PI));
+		public static final Pose3d blueApriltagLocationId3 = new Pose3d(new Translation3d(1.02743, 1.071626, 0), new Rotation3d(0, 0, Math.PI));
+		static List<AprilTag> redApriltags = new ArrayList<>(9) ;
+		static {
+			redApriltags.add(new AprilTag(1, redApriltagLocationId1));
+			redApriltags.add(new AprilTag(2,redApriltagLocationId2));
+			redApriltags.add(new AprilTag(3,redApriltagLocationId3));
+		}
+		
+		static List<AprilTag> blueApriltags = new ArrayList<>(9);
 		static {
-			apriltags.add(new AprilTag(1,apriltagLocation));
+			blueApriltags.add(new AprilTag(6,blueApriltagLocationId1));
+			blueApriltags.add(new AprilTag(7,blueApriltagLocationId2));
+			blueApriltags.add(new AprilTag(8,blueApriltagLocationId3));
 		}
-		public static final AprilTagFieldLayout aprilTagFieldLayout = new AprilTagFieldLayout(apriltags,10,10);
+		public static final AprilTagFieldLayout redAprilTagFieldLayout = new AprilTagFieldLayout(redApriltags,10,10);
 		public static final Transform3d initialCamPosition = new Transform3d(new Translation3d(), new Rotation3d());
 		
```

Comment:
there should only be ONE list of aprilTags, maybe one for debugging, any more are redundent